recipe/build.sh

```diff
@@ -1,26 +1,25 @@
 #!/bin/bash
 
+if [[ $(uname) == 'Darwin' ]]; then
```

Comment:
Still there is the problem that some people recommend to set that variable only for the `make check`, so maybe the original pattern is not that bad.